Download.Mvc/Client/components/FolderView.tsx

```diff
@@ -14,23 +23,53 @@ export class FolderView extends React.Component<IProps, any> {
     constructor(props) {
         super(props);
 
+        this.getLicenseID = this.getLicenseID.bind(this);
+        this.getReadmeID = this.getReadmeID.bind(this);
```

Comment:
@laholstege AFAIK you don't need these lines, ES6 style arrow functions are bound to the proper this scope by default